lua/dim.lua

```diff
@@ -78,8 +78,8 @@ function util.get_treesitter_hl(row, col)
   local buf = vim.api.nvim_get_current_buf()
 
   -- NOTE: use new functionality from https://github.com/neovim/neovim/issues/14090#issuecomment-1228444035
-  if vim.treesitter.get_captures_at_position ~= nil then
```

Comment:
May be good to keep both `get_captures_at_position` and `get_captures_at_pos` for backwards compatibility.

Reply:
Thanks for the suggestion

As far as I can tell, `get_captures_at_position` was introduced in https://github.com/neovim/neovim/commit/b04ef7f6b966c44b12dbc65b17a761ae9313d6c4 which is only included in v0.8.0. In other words, `get_captures_at_position` was a function that existed only in nightly builds before v0.8.0 (v0.7.2 was [released in June](https://github.com/neovim/neovim/releases/tag/v0.7.2)).

IMO it is not worth being backwards compatible with an older nightly version. It makes sense to be backwards compatible with older **stable** versions, but, as far as I checked, this is not the case here

Reply:
Ahh apologies for the confusion! Hope to see this merged soon then :))